is_leaf assertion

```jl
julia> using Revise, Zygote, Debugger
[ Info: Recompiling stale cache file C:\Users\Kristoffer\.julia\compiled\v1.1\Zygote\4kbLI.ji for Zygote [e88e6eb3-aa80-5325-afca-941959d7151f]

julia> l, back = Zygote.forward(x -> sum(x), ones(10,10,10))
(1000.0, getfield(Zygote, Symbol("##71#72")){typeof(∂(getfield(Main, Symbol("##5#6"))()))}(∂(getfield(Main, Symbol("##5#6"))())))

julia>  @enter back(1f0)
Zygote, pc = 14
  (::getfield(Core, Symbol("#kw#Type")))(::Any, ::Type{Zygote.Adjoint}, ir::Core.Compiler.IRCode) in Zygote
    #Adjoint#68(varargs, ::Any, ir::Core.Compiler.IRCode) in Zygote at C:\Users\Kristoffer\.julia\packages\Zygote\6Iu6p\src\compiler\reverse.jl:440
      (::getfield(Core, Symbol("#kw#Type")))(::Any, ::Type{Zygote.Primal}, ir::Core.Compiler.IRCode) in Zygote
        #Primal#42(varargs, ::Any, ir::Core.Compiler.IRCode) in Zygote at C:\Users\Kristoffer\.julia\packages\Zygote\6Iu6p\src\compiler\reverse.jl:228
          record_globals!(ir::Core.Compiler.IRCode) in Zygote at C:\Users\Kristoffer\.julia\packages\Zygote\6Iu6p\src\compiler\reverse.jl:116
            getindex(ir::Core.Compiler.IRCode, a...) in IRTools.Wrap at C:\Users\Kristoffer\.julia\packages\IRTools\IHXm6\src\ir\wrap.jl:14
node = :(return %13)
┌ Warning: error evaluating in module Zygote: #= C:\Users\Kristoffer\.julia\packages\Zygote\6Iu6p\src\precompile.jl:15 =# @code_adjoint pow(2, 3)
│   exception =
│    AssertionError: is_leaf(frame)
│    Stacktrace:
│     [1] step_expr!(::Any, ::JuliaInterpreter.Frame, ::Any, ::Bool) at C:\Users\Kristoffer\.julia\packages\JuliaInterpreter\T2Nfv\src\interpret.jl:411
│     [2] #methods_by_execution!#12(::Bool, ::Bool, ::Function, ::Any, ::Revise.CodeTrackingMethodInfo, ::Dict{Module,Array{Expr,1}}, ::JuliaInterpreter.Frame) at C:\Users\Kristoffer\.julia\packages\Revise\17Klc\src\lowered.jl:173
│     [3] (::getfield(Revise, Symbol("#kw##methods_by_execution!")))(::NamedTuple{(:define,),Tuple{Bool}}, ::typeof(Revise.methods_by_execution!), ::Function, ::Revise.CodeTrackingMethodInfo, ::Dict{Module,Array{Expr,1}}, ::JuliaInterpreter.Frame) at .\none:0
│     [4] #methods_by_execution!#9(::Base.Iterators.Pairs{Symbol,Bool,Tuple{Symbol},NamedTuple{(:define,),Tuple{Bool}}}, ::Function, ::Any, ::Revise.CodeTrackingMethodInfo, ::Dict{Module,Array{Expr,1}}, ::Module, ::Expr) at C:\Users\Kristoffer\.julia\packages\Revise\17Klc\src\lowered.jl:46
│     [5] #methods_by_execution! at .\none:0 [inlined]
│     [6] #eval_with_signatures#57(::Bool, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Function, ::Module, ::Expr) at C:\Users\Kristoffer\.julia\packages\Revise\17Klc\src\Revise.jl:344
│     [7] #eval_with_signatures at C:\Users\Kristoffer\.julia\packages\Revise\17Klc\src\Revise.jl:0 [inlined]
│     [8] #instantiate_sigs!#58(::Bool, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Function, ::OrderedCollections.OrderedDict{Module,OrderedCollections.OrderedDict{Revise.RelocatableExpr,Union{Nothing, Array{Any,1}}}})
at C:\Users\Kristoffer\.julia\packages\Revise\17Klc\src\Revise.jl:352
│     [9] instantiate_sigs! at C:\Users\Kristoffer\.julia\packages\Revise\17Klc\src\Revise.jl:349 [inlined]
│     [10] maybe_parse_from_cache!(::Revise.PkgData, ::String) at C:\Users\Kristoffer\.julia\packages\Revise\17Klc\src\pkgs.jl:224
│     [11] get_def(::Method, ::Revise.PkgData, ::String) at C:\Users\Kristoffer\.julia\packages\Revise\17Klc\src\Revise.jl:748
│     [12] #get_def#66(::Set{Tuple{Revise.PkgData,String}}, ::Function, ::Method) at C:\Users\Kristoffer\.julia\packages\Revise\17Klc\src\Revise.jl:732
│     [13] get_def(::Method) at C:\Users\Kristoffer\.julia\packages\Revise\17Klc\src\Revise.jl:699
│     [14] #invokelatest#1 at .\essentials.jl:742 [inlined]
│     [15] invokelatest at .\essentials.jl:741 [inlined]
│     [16] whereis(::Method) at C:\Users\Kristoffer\.julia\packages\CodeTracking\G9WL2\src\CodeTracking.jl:55
│     [17] locinfo(::JuliaInterpreter.Frame) at C:\Users\Kristoffer\Debugging\Debugger\src\locationinfo.jl:13
│     [18] #print_status#57(::Bool, ::Function, ::Base.TTY, ::JuliaInterpreter.Frame) at C:\Users\Kristoffer\Debugging\Debugger\src\printing.jl:107
│     [19] (::getfield(Debugger, Symbol("#kw##print_status")))(::NamedTuple{(:force_lowered,),Tuple{Bool}}, ::typeof(Debugger.print_status), ::Base.TTY, ::JuliaInterpreter.Frame) at none:0
│     [20] #RunDebugger#6(::Bool, ::Function, ::JuliaInterpreter.Frame, ::Nothing, ::Nothing) at C:\Users\Kristoffer\Debugging\Debugger\src\repl.jl:135
│     [21] RunDebugger at C:\Users\Kristoffer\Debugging\Debugger\src\repl.jl:4 [inlined] (repeats 2 times)
│     [22] top-level scope at C:\Users\Kristoffer\Debugging\Debugger\src\Debugger.jl:107
│     [23] eval(::Module, ::Any) at .\boot.jl:328
│     [24] eval_user_input(::Any, ::REPL.REPLBackend) at C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.1\REPL\src\REPL.jl:85
│     [25] run_backend(::REPL.REPLBackend) at C:\Users\Kristoffer\.julia\packages\Revise\17Klc\src\Revise.jl:848
│     [26] (::getfield(Revise, Symbol("##70#72")){REPL.REPLBackend})() at .\task.jl:259
└ @ Revise C:\Users\Kristoffer\.julia\packages\Revise\17Klc\src\lowered.jl:49
┌ Warning: omitting call expression (getfield(Core, Symbol("#kw#Type"))())((varargs = nothing,), Zygote.Adjoint, 17 1 ─ %1 = (Base.broadcasted)(Zygote.:*, _2, _3)::Any
│    │   %2 = (Base.materialize)(%1)::Any
│    │   %3 = (Zygote.sum)(%2)::Any
│    └──      return %3
│ ) in ("none", 0)
└ @ Revise C:\Users\Kristoffer\.julia\packages\Revise\17Klc\src\lowered.jl:166
```

--- a/src/lowered.jl
+++ b/src/lowered.jl
@@ -61,6 +61,7 @@ function methods_by_execution!(@nospecialize(recurse), methodinfo, docexprs, fra
     signatures = []  # temporary for method signature storage
     pc = frame.pc
     while true
+        JuliaInterpreter.is_leaf(frame) || break
         stmt = pc_expr(frame, pc)
         if isa(stmt, Expr)
             if stmt.head == :struct_type || stmt.head == :abstract_type || stmt.head == :primitive_type

